adcc/backends/psi4.py

```diff
@@ -53,6 +54,17 @@ def magnetic_dipole(self):
     def nabla(self):
         return [-1.0 * np.asarray(comp) for comp in self.mints.ao_nabla()]
 
+    @property
+    def density_dependent_operators(self):
+        ret = {}
+        if hasattr(self.wfn, "pe_state"):
+            ret["pe_induction_elec"] = lambda dm: \
```

Comment:
IMHO if you need to break a lambda make it a def

Reply:
Done.